src/signals/incident/components/form/ContainerSelect/WfsLayer/context.tsx

```diff
@@ -0,0 +1,22 @@
+import type { FunctionComponent } from 'react';
+import React from 'react';
+import type { FeatureCollection } from 'geojson';
+
+export const NO_DATA: FeatureCollection = {
+  type: 'FeatureCollection',
+  features: [],
+};
+
+
+const initialValue: FeatureCollection = NO_DATA;
+
+const WfsDataContext = React.createContext(initialValue);
+
+interface WfsDataProviderProps {
+  value: FeatureCollection;
+}
+
+export const WfsDataProvider: FunctionComponent<WfsDataProviderProps> = ({ value, children }) =>
```

Comment:
What's the use of this?

Reply:
Besides the `WfsDataContext` exported on the line below.

Reply:
I prefer to use all Context related stuff in one file and `<[ContextName].Provider /> ` construction in one file. This is the suggested way I found in few of the `Pluralsight` courses I followed. See for refference: 'https://www.pluralsight.com/guides/how-to-use-react-context-to-share-data-between-components'

Reply:
You're not using it the way they explain in that article, though.
`WfsDataProvider` here is just an empty shell that doesn't add any functionality.
In your component, you can just as well use `WfsDataContext.Provider`.

Reply:
True, I am not passing any setter through because at this moment is not needed but this might change in the future. Of course I could use the `WfsDataContext.Provider` but I consider that less readable because you import a component and use a subcomponent unlike the rest of the code where you would destructure it direct `import { Provider } from './WfsDataContext'`. This breaks in my opinion the coding guidelines we are currently using.

Reply:
I'm not sure. I would expect, in this example, to see `WfsDataContext` and `WfsDataContext.Provider` being used. Using the pattern you are using, you deviate from the default approach and naming, and thus from the expectations. This creates confusion in my opinion.
Having said that, if you intend to keep using this pattern, you can, in this case, write it like this:

```
export const WfsDataProvider: FunctionComponent<WfsDataProviderProps> = WfsDataContext.Provider;
```